.github/workflows/dev.yml

```diff
@@ -116,6 +116,8 @@ jobs:
 
       - name: "set up compile-env"
         run: |
+          docker run --privileged --rm busybox:latest sh -c "echo 512 > /sys/kernel/mm/hugepages/hugepages-2048kB/nr_hugepages && cat /sys/kernel/mm/hugepages/hugepages-2048kB/nr_hugepages"
+          docker pull ghcr.io/githedgehog/testn/n-vm:0.0.5
```

Comment:
These Docker commands are not really setting up the `compile-env` image, I'd move them to separate steps; or rename this step

Reply:
resolved I think